controllers/monitoring_alert.go

```diff
@@ -180,7 +213,17 @@ func (r *ManagedMCGReconciler) reconcileMonitoringResources() error {
 	}
 	for i := range promRuleList.Items {
 		obj := promRuleList.Items[i]
-		utils.AddLabel(obj, monLabelKey, monLabelValue)
+		// do not include the original noobaa rules (deployed by mcg-operator)
+		// in the rule files set that our external prometheus instance will load
+		if obj.GetName() == "noobaa-prometheus-rules" {
+			r.Log.Info("found noobaa prometheus rules, checking for inclusion")
+			if v, ok := obj.GetLabels()[monLabelKey]; ok && v == monLabelValue {
+				utils.RemoveLabel(obj, monLabelKey)
```

Comment:
one doubt, we are removing the original Prometheus rule to add the labels origin: mcg-osd-deployer which will help us to identify our metrics? if yes where are we adding this labels to metrics? @rexagod

Reply:
We are removing the original rule file to remove/add/edit a bunch of custom rules and alerts, in order to provide clarity to the user. We are adding these labels here: https://github.com/red-hat-storage/mcg-osd-deployer/pull/61/files#diff-c2d8ea863a8b41a65ff336525a37d8a9b198d683b1dae92c7d4be6f06c446d7eR7.